.github/workflows/build.yml

```diff
@@ -43,6 +43,14 @@ jobs:
     steps:
     - uses: actions/checkout@v1
 
+    # Set Path workaround for https://github.com/actions/virtual-environments/issues/263
+    - run: |
+        echo "::add-path::C:\Program Files\Git\mingw64\bin"
```

Comment:
Do we have a guarantee that git is installed in this path? Should we build the path based on `%PROGRAMFILES%` instead?

Reply:
The PR that broke this was here: https://github.com/actions/virtual-environments/pull/211/files

So I just manually added these paths back. I don't envision this being a long term solution so I feel something a little hacky like this should be okay.

Reply:
Ah, the virtual environment makes the same assumptions:

https://github.com/actions/virtual-environments/pull/211/files

So we should be good with these changes then.